apps/supernova/src/types/index.d.ts

```diff
@@ -0,0 +1,7 @@
+/*
+taymoor89 marked this conversation as resolved.
```

Comment:
hmm, that line shouldn't be in the SPDX section

Reply:
haha my fault